internal/runner/abstract_manager.go

```diff
@@ -58,7 +58,17 @@ func (n *AbstractManager) DeleteEnvironment(id dto.EnvironmentID) {
 }
 
 func (n *AbstractManager) EnvironmentStatistics() map[dto.EnvironmentID]*dto.StatisticalExecutionEnvironmentData {
-	return map[dto.EnvironmentID]*dto.StatisticalExecutionEnvironmentData{}
+	environments := make(map[dto.EnvironmentID]*dto.StatisticalExecutionEnvironmentData)
+	for _, e := range n.environments.List() {
+		environments[e.ID()] = &dto.StatisticalExecutionEnvironmentData{
+			ID:                 int(e.ID()),
+			PrewarmingPoolSize: e.PrewarmingPoolSize(),
+			IdleRunners:        uint(e.IdleRunnerCount()),
+			UsedRunners:        0,
```

Comment:
I noticed that these lines were moved from the Nomad manager, but why are we returning 0 for the `UsedRunenrs` here? These are added later as the information is present in the specific runner manager?

Reply:
Exactly! Would you like a comment on this?

Reply:
Yes, that would be great but think it is optional 👒

Reply:
```suggestion
			UsedRunners:        0, // Adjusted later as the information is present in the specific runner manager.
```